src/library/Uno.Themes.WinUI.Markup/Theme/ToggleButton.cs

```diff
@@ -0,0 +1,379 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushCheckedPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("TextToggleButtonBorderBrushCheckedPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushCheckedPressed")]
+						public static ResourceValue<Brush> Pressed => new("TextToggleButtonBorderBrushCheckedPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushCheckedDisabled")]
+						public static ResourceValue<Brush> Disabled => new("TextToggleButtonBorderBrushCheckedDisabled", true);
+					}
+
+					public static class BorderBrushIndeterminate
+					{
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushIndeterminate")]
+						public static ResourceValue<Brush> Default => new("TextToggleButtonBorderBrushIndeterminate", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushIndeterminatePointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("TextToggleButtonBorderBrushIndeterminatePointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushIndeterminatePressed")]
+						public static ResourceValue<Brush> Pressed => new("TextToggleButtonBorderBrushIndeterminatePressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushIndeterminateDisabled")]
+						public static ResourceValue<Brush> Disabled => new("TextToggleButtonBorderBrushIndeterminateDisabled", true);
+					}
+
+					public static class Feedback
+					{
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonFeedbackFocused")]
+						public static ResourceValue<Brush> Focused => new("TextToggleButtonFeedbackFocused", true);
+					}
+
+					public static class Typography
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "TextToggleButtonFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("TextToggleButtonFontFamily", true);
+
+						[ResourceKeyDefinition(typeof(double), "TextToggleButtonFontSize")]
+						public static ResourceValue<double> FontSize => new("TextToggleButtonFontSize", true);
+
+						[ResourceKeyDefinition(typeof(Thickness), "TextToggleButtonBorderThickness")]
+						public static ResourceValue<Thickness> BorderThickness => new("TextToggleButtonBorderThickness", true);
+
+						[ResourceKeyDefinition(typeof(CornerRadius), "TextToggleButtonCornerRadius")]
+						public static ResourceValue<CornerRadius> CornerRadius => new("TextToggleButtonCornerRadius", true);
+
+						[ResourceKeyDefinition(typeof(Thickness), "TextToggleButtonPadding")]
+						public static ResourceValue<Thickness> Padding => new("TextToggleButtonPadding", true);
+
+						[ResourceKeyDefinition(typeof(double), "TextToggleButtonMinHeight")]
+						public static ResourceValue<double> MinHeight => new("TextToggleButtonMinHeight", true);
```

Comment:
These don't belong in the `Typography` class, I believe these can simply be properties in the parent `Text`/`Icon` class 

Right, @agneszitte / @dansiegel ?